server/test/api/references.js

```diff
@@ -24,50 +25,50 @@ describe('The reference API, PATH : /references', function () {
     'id', 'is_report', 'ref', 'text', 'position', 'reference_group_id', 'section_resultat_id'
   ];
 
-  before(helpers.login(agent));
+  beforeEach(helpers.login(agent));
```

Comment:
In this pull request https://github.com/IMA-WorldHealth/bhima-2.X/pull/237 @jniles updated many of the integration tests to use `before` instead of `befereEach`; we found that this almost [doubled performance](https://github.com/IMA-WorldHealth/bhima-2.X/pull/237#issuecomment-203879169). 

This would be a good pattern to use going forward.